.github/workflows/ci.yml

```
@@ -0,0 +1,28 @@
+name: "CI"
+
+on:
+  push:
+    branches: ['main']
+  pull_request:
+    branches: ['main']
+jobs:
+  tls:
+    runs-on: ubuntu-latest    
+    steps:
+      - uses: actions/checkout@v2
+      - name: install dependencies
+        run: sudo apt update && sudo apt install -y rapidjson-dev
+      - name: build
+        run: sh ./build.sh
+      - name: prepare models
+        run: cd examples && . fetch_model.sh && cd ..
```

I think use of source or . is easier to read/scan.

```suggestion
        run: cd examples && source fetch_model.sh && cd ..
```

---

👍